import_test.go

```diff
@@ -160,7 +161,7 @@ func TestImporter_Close(t *testing.T) {
 	importer, err := tree.Import(1)
 	require.NoError(t, err)
 
-	err = importer.Add(&ExportNode{Key: []byte("key"), Value: []byte("value"), Version: 1, Height: 0})
+	err = importer.Add(&ExportNode{Key: []byte("key"), Value: []byte("value"), NodeKey: &NodeKey{version: 1, path: big.NewInt(1)}, Height: 0})
```

Comment:
how does this work for existing exports?or is the migration path planned to be different ?

Reply:
I believe it would look different, please refer #662